docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -125,34 +125,40 @@ \section{Failure Mode and Effect Analysis}
       site, but unable to follow entry/exit protocols & a. GPS
       Signal Blocked \newline b. Use of inaccurate Geolocation API
       \newline c. Out of date third-party databases \newline d.
-      Device does not have a GPS \newline e. VPN usage & Error
-      messages and logging & a. Report inaccurate location detection
-      to manager & SR-AR1 & H1-1 \\
+      Device does not have a GPS \newline e. VPN usage & System
+      provides error message to user when they get geo-blocked and
+      sends an email to manager indicating that the user has been
+      geo-blocked & a. User reports inaccurate location detection to
+      manager & SR-AR1 & H1-1 \\
```

Comment:
Just interested these Safety Requirements SR-AR1 etc have been mentioned here but aren't actually described in the next section. Are we planning to this later?

Reply:
oh those are pre-existing ones that I've mentioned here. I think the next section is for new ones only

Reply:
So these have already been mentioned in the SRS?

Reply:
Yeah they are